scripts/metadata_comparison/metadata_comparison/comparer.py

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env python3
+#
+# comparer.py
+#
+# Purpose: Compare performance metadata JSON files produced by Digester and produce result in CSV format
+#
+# Usage: python3 comparer.py [-h] [-v] [--json_paths JSONPATH [JSONPATH ...]] [--output_path OUTPUTPATH]
+#
+# Python Prereqs (at least, the ones which I needed to manually install... YMMV):
+#
+#   * pip3 install --upgrade pandas
+#
+
+import argparse
+import json
+import pandas
+from pathlib import Path
+import logging as log
+
+logger = log.getLogger('metadata_comparison.comparer')
+
+def set_log_verbosity(verbose):
```

Comment:
Maybe extract this to a lib rather than copy/pasting the same function?

Reply:
yeah I did that in my embryonic branch...